app/interfaces/api/entities/claim.rb

```diff
@@ -30,11 +30,11 @@ class Claim < BaseEntity
       private
 
       def messages_count
-        object.messages.count
+        object.messages.size
       end
 
       def unread_messages_count(user)
-        object.unread_messages_for(user).count
+        object.unread_messages_for(user).size
```

Comment:
This was supposed to behave the same way as [messages_count](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2316/files#r180154528), but unfortunately due the having a specific user scope, I was unable to preload this :S.